.circleci/config.yml

```diff
@@ -8,6 +8,7 @@ references:
       command: |
         cd server
         pipenv install
+        export FLASK_APP=mentionscrawler
```

Comment:
You can also set env vars for circle ci on https://circleci.com/gh/hatchways/team-ginger/edit#env-vars
Useful for sensitive data like secrets and api keys